sdk/search/search-documents/test/node/searchIndexerClient.spec.ts

```diff
@@ -0,0 +1,264 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/* eslint-disable no-invalid-this */
+import {isPlaybackMode, record, Recorder} from "@azure/test-utils-recorder";
+import {assert} from "chai";
+import {SearchIndexClient, SearchIndexerClient, SearchIndexerDataSourceConnection, SearchIndexerSkillset, SearchIndexer} from "../../src/index";
+import {Hotel} from "../utils/interfaces";
+import {createClients, environmentSetup} from "../utils/recordedClient";
+import {createIndex, createDataSourceConnections, deleteDataSourceConnections, createSkillsets, deleteSkillsets, createIndexers, deleteIndexers} from "../utils/setupIndex";
+import { delay } from "@azure/core-http";
+
+const TEST_INDEX_NAME = "hotel-live-test2";
+
+describe("SearchIndexerClient", function() {
+  let recorder: Recorder;
+  let indexerClient: SearchIndexerClient;
+  let indexClient: SearchIndexClient;
+
+  this.timeout(99999);
+
+  beforeEach(async function() {
+    ({ indexClient, indexerClient } = createClients<Hotel>(TEST_INDEX_NAME));
+    if (!isPlaybackMode()) {
+      await createDataSourceConnections(indexerClient);
+      await createSkillsets(indexerClient);
+      await createIndex(indexClient, TEST_INDEX_NAME);
+      await delay(5000);
+      await createIndexers(indexerClient);
+    }
+    recorder = record(this, environmentSetup);
+    // create the clients again, but hooked up to the recorder
+    ({ indexClient, indexerClient } = createClients<Hotel>(TEST_INDEX_NAME))
+  });
+
+
+  afterEach(async function() {
+    if (recorder) {
+      await recorder.stop();
+    }
+    if (!isPlaybackMode()) {
+      await indexClient.deleteIndex(TEST_INDEX_NAME);
+      await delay(2000);
+      await deleteDataSourceConnections(indexerClient);
+      await deleteSkillsets(indexerClient);
+      await deleteIndexers(indexerClient);
+    }
+  });
+
+  describe("#indexers", function() {
+    it("gets the list of indexers", async function(){
+      const indexers = await indexerClient.listIndexers();
+      assert.equal(indexers.length >= 5, true);
```

Comment:
use assert.isAtLeast: https://www.chaijs.com/api/assert/#method_isatleast

Reply:
Thanks. Modified it